### PR TITLE
Updated to support Julia 1.0+, removed support for Julia 0.6

### PR DESCRIPTION
**Pull request by mattBrzezinski**
Summary of changes
---
- Added support for Julia 1.0+, dropped support for Julia 0.6
- Added Julia 1.1 CI build for Appveyor and Travis. Removed 0.6, 0.7 builds for Travis
- Removed commented CSS code in `docs/src/assets/invenia.css`

Testing
---
- Built the documentation with the same deployment results as the `Dispatcher.jl` project
- Ran the unit tests successfully without any warnings

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 environment:
   matrix:
   - julia_version: 1.0
+  - julia_version: 1.1
```
Comment (iamed2) [inline .appveyor.yml]:
```suggestion
  - julia_version: 1.1
  - julia_version: 1.2
```

File: .travis.yml [hunk 2]
```diff
@@ -4,9 +4,8 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
-  - 0.7
   - 1.0
+  - 1.1
```
Comment (iamed2) [inline .travis.yml]:
```suggestion
  - 1.1
  - 1.2
```

File: Project.toml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+uuid = "9c2b0ca7-0fb3-5789-8f1a-7604e426024c"
+
+[deps]
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
+PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+
+[compat]
+julia = "≥ 1.0.0"
```
Comment (iamed2) [inline Project.toml]:
```suggestion
julia = "1"
```
We can't claim it'll run on Julia 2+

File: .travis.yml (hunk 2 above)
Comment (iamed2) [reply .travis.yml]: 1.2 will be released soon, but it's already available on CI platforms

File: REQUIRE [hunk 4]
```diff
@@ -1,5 +1,4 @@
-julia 0.6
-Compat
+julia 1.0
```
Comment (iamed2) [inline REQUIRE]: This whole file should be deleted now

File: Project.toml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+name = "FTPServer"
+uuid = "9c2b0ca7-0fb3-5789-8f1a-7604e426024c"
```
Comment (oxinabox) [inline Project.toml]: Add a version here (check on GitHub releases tab and increment)
Add `author="Invenia Technical ....`

File: Project.toml (hunk 3 above)
Comment (oxinabox) [inline Project.toml]: Don't use lower bounds, use the default (caret) bounds. See Pkg docs
Prob add compat for all packages too here.

**Comment by iamed2** [comment]
Here's a good docs example: https://github.com/invenia/Impute.jl/blob/master/.travis.yml#L21

**Comment by iamed2** [comment]
Add the deploy key following the instructions listed [here](https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#SSH-Deploy-Keys-1) and we're good to go

**Comment by mattBrzezinski** [comment]
Added the deploy key for Travis. :+1:

**Pull request merged**